src/shared/component/SwaggerURLInput.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import './SwaggerURLInput.scss';
+
+class SwaggerURLInput extends React.Component {
+
+  onSubmit(e) {
+    e.preventDefault();
+
+    const swaggerURL = this.swaggerURLInput.value;
```

Comment:
I've never used this way of handling forms, mostly because it's not recommended. It is specified in the documentation page that it is an escape hatch. They call that uncontrolled components and the recommendation is to use controlled components: https://facebook.github.io/react/docs/forms.html
I'm not saying this is wrong, and it totally works, so you don't have to go through the trouble of converting this field to a controlled component. It's just cleaner and it respects React data flow better.